### PR TITLE
add appliance tests

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
We run this tests in a jenkins job against an appliance that has already
been launched on an openstack cloud.

### CODE REVIEW

**Comment by flavio** [comment]
LGTM

File: packaging/suse/appliance-test/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Use this scripts to test the appliance from a jenkins job.
```
Comment (mssola) [inline packaging/suse/appliance-test/README.md]: nitpicking: "scripts" -> "script", since there's just one script ;)

**Comment by mssola** [comment]
LGTM :+1: As said on IRC, you should add this here: https://github.com/SUSE/Portus/wiki/How-we-test-Portus#the-appliance

**Comment by jordimassaguerpla** [comment]
fixed the README

**Pull request merged**